.travis.yml

```diff
@@ -3,11 +3,11 @@ language: node_js
 node_js:
 - 8
 before_install:
-- npm install codecov.io coveralls
+- npm install codecov coveralls
 - npm install nwb
 after_success:
-- cat ./coverage/lcov.info | ./node_modules/codecov.io/bin/codecov.io.js
-- cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js
+- cat ./coverage/lcov.info | ./node_modules/.bin/codecov --pipe
+GuillaumeAmat-patch-1- cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js
```

Comment:
Could you explain this change to me please? I don't understand why there's a reference to your patch branch in there

Reply:
Woops... No reason at all!... Sorry :/

I'm fixing it.

Reply:
Done @embiem